manual/manual/library/libnum.etex

```diff
@@ -1,6 +1,67 @@
-\chapter{The num library: arbitrary-precision rational arithmetic}
+\chapter{Recently removed or moved libraries (Graphics, Bigarray, Num, LablTk)}
 %HEVEA\cutname{libnum.html}
 
+This chapter describes three libraries which were formerly part of the OCaml
+distribution (Graphics, Num, and LablTk), and a library which has now become
+part of OCaml's standard library, and is documented there (Bigarray).
+
+
+\section{s:graphics-removed}{The Graphics Library}
+
+Since OCaml 4.09, the "graphics" library is distributed as an external
+package. Its new home is:
+
+\url{https://github.com/ocaml/graphics}
+
+If you are using the opam package manager, you should install the
+corresponding "graphics" package:
+
+\begin{alltt}
+        opam install graphics
+\end{alltt}
```

Comment:
These new additions to `libnum.etex` look like freshly created content in the git patchset, there is no corresponding deletion. I suspect that you forgot to tell git that the other files are removed.

Reply:
Ditto.